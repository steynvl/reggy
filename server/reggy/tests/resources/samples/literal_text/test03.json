{
  "sampleStringsInfo": [
    [
      {
        "markerType": "Literal text",
        "markedStrings": [
          "jannis"
        ],
        "markerInfo": {
          "caseInsensitive": false,
          "matchAllExceptSpecified": false
        },
        "repeatInfo": {
          "repeat": "1"
        }
      }
    ],
    [
      {
        "markerType": "Literal text",
        "markedStrings": [
          "kannis"
        ],
        "markerInfo": {
          "caseInsensitive": false,
          "matchAllExceptSpecified": false
        },
        "repeatInfo": {
          "repeat": "1"
        }
      }
    ],
    [
      {
        "markerType": "Literal text",
        "markedStrings": [
          "\""
        ],
        "markerInfo": {
          "caseInsensitive": false,
          "matchAllExceptSpecified": false
        },
        "repeatInfo": {
          "repeat": "1"
        }
      }
    ]
  ],
  "generalRegexInfo": {
    "startRegexMatchAt": "Anywhere",
    "endRegexMatchAt": "Anywhere",
    "regexTarget": "Java"
  },
  "generateMethod": "samplesAndSemantics"
}
