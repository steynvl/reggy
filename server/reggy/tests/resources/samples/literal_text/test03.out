Java
Pattern regex = Pattern.compile("janniskannis\"");
janniskannis\"
Perl
my $regex = /janniskannis"/;
janniskannis"
POSIX
janniskannis"
janniskannis"
Python
regex = re.compile(r'janniskannis"')
janniskannis"
JavaScript
const regex = /janniskannis"/;
janniskannis"
PHP
$regex = '/janniskannis"/';
janniskannis"
Golang
regex, _ := regexp.Compile("janniskannis\"")
janniskannis\"
Rust
let re = Regex::new(r"janniskannis\"").unwrap();
janniskannis\"
C#
var re = new Regex(@"janniskannis\"");
janniskannis\"
Scala
val re = "janniskannis\"".r
janniskannis\"
Kotlin
val regex = Regex("janniskannis\"")
janniskannis\"